packages/@ama-sdk/schematics/schematics/typescript/shell/templates/base/openapitools.json

```diff
@@ -3,6 +3,13 @@
   "spaces": 2,
   "generator-cli": {
     "version": "<%= openApiSupportedVersion %>",
-    "storageDir": ".openapi-generator"
+    "storageDir": ".openapi-generator",
+    "generators": {
+      "<%=projectName%>-<%=projectPackageName%>": {
```

Comment:
We should also change the package.json generated to call the schematics with `--generator-key` instead